treeherder/webapp/api/perf_compare_utils.py

```diff
@@ -70,12 +70,21 @@ def get_avg(values, header):
     @param header: name of the header
     @return: Mean of the runs values if there are any
     """
+
     if header == NOISE_METRIC_HEADER:
         return get_noise_metric_avg(values)
     else:
         return mean(values) if len(values) else 0
 
 
+def get_median(values):
+    """
+    @param values: list of the runs values
+    @return: Median of the runs values if there are any
+    """
+    return median(values) if len(values) else 0
```

Comment:
@kimberlythegeek do you know if the median value should be calculated differently if the test is a 'noise metric'? The mean value is calculated differently on this case.

Reply:
I'm not sure. I asked sparky and this was his response was no: "The main reason I say no is because I don't think anyone uses that metric much, if at all, either"

he's referring to these metrics: https://treeherder.mozilla.org/perfherder/compare?originalProject=mozilla-central&originalRevision=09bcd3a0096b6d38671c7da6c6980dd9a7e373bb&newProject=mozilla-central&newRevision=3ac703c87e3c7e1c9a1e8ad85722589d1dbf1a14&framework=13&page=1&filter=noise+metric

I'm not familiar with noise metrics--is this the answer you needed?

Reply:
Yes, thanks!